receiver/filelogreceiver/README.md

```diff
@@ -23,16 +23,16 @@ Tails and parses logs from files.
 | `exclude_older_than`                |                                      | Exclude files whose modification time is older than the specified [age](#time-parameters).                                                                                                                                                                      |
 | `start_at`                          | `end`                                | At startup, where to start reading logs from the file. Options are `beginning` or `end`.                                                                                                                                                                        |
 | `multiline`                         |                                      | A `multiline` configuration block. See [below](#multiline-configuration) for more details.                                                                                                                                                                      |
-| `force_flush_period`                | `500ms`                              | [Time](#time-parameters) since last read of data from file, after which currently buffered log should be send to pipeline. A value of `0` will disable forced flushing.                                                                                         |
+| `force_flush_period`                | `500ms`                              | [Time](#time-parameters) since last token was emitted from the file, after which a partial log at the end of the file may be emitted.|
```

Comment:
```suggestion
| `force_flush_period`                | `500ms`                              | [Time](#time-parameters) since last time new data was found in the file, after which a partial log at the end of the file may be emitted.|
```